alphabase/constants/const_files/psm_reader.yaml

```diff
@@ -262,3 +265,4 @@ sage:
     'peptide_fdr': 'peptide_q'
     'protein_fdr': 'protein_q'
     'decoy': 'is_decoy'
+  modification_mapping_type: 'maxquant'
```

Comment:
I'm not sure if this has an effect. sage uses a different mapping method to my knowledge.